custom/staking/src/lib.rs

```diff
@@ -807,6 +810,16 @@ decl_storage! {
                         <Module<T>>::validate(
                             T::Origin::from(Some(controller.clone()).into()),
                             Default::default(),
+                        ).ok();
+                        <Module<T>>::add_potential_validator(
+                            // TODO: change origin to committee
+                            frame_system::RawOrigin::Root.into(),
+                            stash.clone()
+                        ).ok();
+                        <Module<T>>::compliance_passed(
+                            // TODO: change origin to committee
+                            frame_system::RawOrigin::Root.into(),
+                            stash.clone()
```

Comment:
isn't `controller.clone()` instead of `stash.clone()` ?

Reply:
No, it's the stash acccounts. Tests are written that way as well.

Reply:
ahh make sense after checking the actual code, I was confused with signature of `add_potential_validator()` function i.e 
```
fn add_potential_validator(origin, controller: T::AccountId) 
```
I think it is better to change the `controller` name to `stash` for consistency.  same applies for `remove_validator` ,`compliance_failed` & `compliance_passed`

Reply:
Yeah - the arg name should have been `validator`. Renamed args.